### PR TITLE
Made in comparison + detailed comparison + selectors

### PR DESCRIPTION
**Pull request by Code-Hello**
page made in comparison started
page detailed comparison done
selectors done and added to pages conviction

### CODE REVIEW

**Comment by lebowvsky** [comment]
Page Priorities Done on _Priorities.jsx_

File: src/components/ConvictionsEcology.jsx [hunk 1]
```diff
@@ -51,56 +51,47 @@ const ConvictionsEcology = () => {
         <SectionTitle>Ecologie</SectionTitle>
       </MainHeaderEcology>
       <BurgerMenu />
-      <Cursor
-        titleCursor="Politique environementale"
+      <Selector
+        titleSelect="Politique environementale"
```
Comment (KatalinaNadasi) [inline src/components/ConvictionsEcology.jsx]: non bloquant : petite typo à "environnementale" ==> 2 'n'

File: src/components/DetailedComparisonItem.jsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+              <WrapperCaretIconGrey id={`toRotate${id}`}>
+                <FontAwesomeIcon icon={faCaretRight} />
+              </WrapperCaretIconGrey>
+              <TitleIndicatorDetail>{titleTheme}</TitleIndicatorDetail>
+            </WrapperTitleAccordion>
+          </WrapperHeadAccordion>
+          <FlexDiv>
+            {themesWithNotes.map((themeNote) => {
+              return (
+                <DetailedNoteCountry style={{ width }} borderRight>
```
Comment (KatalinaNadasi) [inline src/components/DetailedComparisonItem.jsx:67]: there is a width without value in the inline style

File: src/styles/containers.jsx [hunk 3]
```diff
@@ -122,6 +138,7 @@ export const CountryCard = styled.div`
   background-color: ${(props) => props.theme.lightGrey};
   border-radius: 10px;
   width: 100%;
+  margin: 20px 0px 20px 0px;
```
Comment (KatalinaNadasi) [inline src/styles/containers.jsx]: You can just write : `margin: 20px 0`

Comment (KatalinaNadasi) [reply src/styles/containers.jsx]: this is the same than margin: `20px 0px 20px 0px`

Comment (KatalinaNadasi) [reply src/styles/containers.jsx]: and no px needed for 0

**Review by KatalinaNadasi — COMMENTED** [review]
Just few comments

File: src/components/DetailedComparisonItem.jsx (hunk 2 above)
Comment (Code-Hello) [reply src/components/DetailedComparisonItem.jsx:67]: This width refers to the width defined at the beginning of the component : " const width = `calc(100%/${nbCountries})`; "

**Review by KatalinaNadasi — APPROVED** [review]

**Pull request merged**